.golangci.yml

```diff
@@ -7,5 +7,17 @@ run:
 linters:
   enable:
     - gofumpt
+    - gosec
     - errorlint
     - unconvert
+
+linters-settings:
+  gosec:
+    confidence: medium
+    severity: medium
```

Comment:
In the PR description you said "we need it pass severity: high" and yet you put it to medium here.

Reply:
See above:

> Hm, it finds issues with confidence: HIGH and severity: MEDIUM, though I filter the other way around. I'll set both to medium and fix or nosec the findings.